smap/sources/conf/twitter_conf.ini

```diff
@@ -0,0 +1,14 @@
+[server]
+Port = 8002
+
+[/]
+uuid = 6fae1782-27b2-12e1-2a48-370ecdac5e01
+
+[/twitter]
+type = twitterDriver.TwitterDriver
+Rate = 43200
+consumer_key = 5EvNnwcByDSdb0rxiW5J0mKaT
+consumer_secret = kBU9LfH099KajuZiRJasYJWI6agX6J0CViTY4QciRptLpfoFPX
```

Comment:
I would avoid checking in the secret keys. With this checkin, people can post fake data on twitter and delete our real data, right? I would regenerate the keys and put the file directly on the server. I'll copy from there into a UC Berkeley internal repository. Or I can give you access to the internal repo if you prefer.